Source/Base/ASBaseDefines.h

```diff
@@ -20,6 +20,14 @@
 #define AS_EXTERN FOUNDATION_EXTERN
 #define unowned __unsafe_unretained
 
+#if defined(__cplusplus)
+# define var auto
+# define let auto const
+#else
+# define var __auto_type
+# define let const __auto_type
```

Comment:
Let's be consistent on ordering: `# define let __auto_type const`

Reply:
I moved the `const` in front of the `auto`. I would say putting the const first is arguably more readable, since it follows English in putting the "adjective" (const) before the "noun" (type).